src/fswAlgorithms/stateEstimation/thrustCMEstimation/thrustCMEstimation.cpp

```diff
@@ -72,5 +72,90 @@ void ThrustCMEstimation::Reset(uint64_t CurrentSimNanos)
  */
 void ThrustCMEstimation::UpdateState(uint64_t CurrentSimNanos)
 {
-    
+    /*! create output message buffers */
+    VehicleConfigMsgPayload vehConfigOutBuffer;
```

Comment:
zero the output message structure on construction.  You can add `= {}` to zero this.

Reply:
Done.